docker-compose.yml

```diff
@@ -74,6 +74,7 @@ services:
       - "${AD_SERVICE_PORT}"
     environment:
       - AD_SERVICE_PORT
+      - FEATURE_FLAG_GRPC_SERVICE_ADDR
```

Comment:
Need to make a corresponding update to the [helm charts](https://github.com/open-telemetry/opentelemetry-helm-charts).